src/assets/css/styles.css

```diff
@@ -1,6 +1,87 @@
 .container-fluid {
-    background-color: #00386c;
+    /*background-color: #00386c;*/
     color: #ffffff;
     display: block;
     height: 100vh;
-}
\ No newline at end of file
+    background-image: url('../img/background.png');
+}
```

Comment:
It looks like `.container-fluid` is no longer used, can we remove these styles?

Reply:
fixed